keyboards/preonic/keymaps/solstice22/config.h

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
```

Comment:
Actually, most of this is not needed.  You should remove everything here that is from the keyboard's config.h, and only include the changes that are NOt present there. 

Which I suspect is just:

```c
#define QMK_ESC_OUTPUT F1
#define QMK_ESC_INPUT B5
#define QMK_LED     E6
#define QMK_SPEAKER C6
```